widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -68,7 +68,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         queueId = intent.getStringExtra(GliaWidgets.QUEUE_ID);
         runtimeTheme = intent.getParcelableExtra(GliaWidgets.UI_THEME);
         contextUrl = intent.getStringExtra(GliaWidgets.CONTEXT_URL);
-        useOverlays = intent.getBooleanExtra(GliaWidgets.USE_OVERLAY, true);
+        useOverlays = intent.getBooleanExtra(GliaWidgets.USE_OVERLAY, Dependencies.getSdkConfigurationManager().isUseOverlay());
```

Comment:
I suggest to use a separate method to get default value, improves readability.

Reply:
Introduced Configuration class - moved stuff around a bit